labs/multiverse/src/widgets/room_view/timeline.rs

```diff
@@ -52,11 +52,51 @@ impl Widget for &mut TimelineView<'_> {
                             ..
                         }) => content.push(format!("{}: (UTD)", sender)),
 
+                        TimelineItemContent::MembershipChange(m) => {
+                            if let Some(change) = m.change() {
+                                let display_name =
+                                    m.display_name().unwrap_or_else(|| m.user_id().to_string());
+
+                                let change = match change {
+                                    MembershipChange::Joined => "has joined the room",
+                                    MembershipChange::Left => "has left the room",
+                                    MembershipChange::Banned => "has been banned",
+                                    MembershipChange::Unbanned => "has been unbanned",
+                                    MembershipChange::Kicked => "has been removed from the room",
+                                    MembershipChange::Invited => "has been invited to the room",
+                                    MembershipChange::KickedAndBanned => {
+                                        "has been removed and banned from the room"
```

Comment:
"kicked and banned"

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4944/commits/a52725b7bb95539d4136d281b4a1e4025d440ddb